src/components/NewsletterForm/NewsletterForm.tsx

```diff
@@ -26,18 +81,21 @@ export const NewsletterForm = ({
       method="POST"
       onSubmit={handleSubmit}
     >
-      {/* The hidden inputs below are required by dotmailer */}
```

Comment:
As we'll be submitting to the D7 site, we won't need the hidden inputs as the server (D7) will handle these.